src/object/cli_ec.c

```diff
@@ -57,27 +57,67 @@ struct ec_params {
 	struct ec_params        *next;	/* Pointer to next entry in list. */
 };
 
+struct ec_fetch_params {
+	daos_iod_t		*iods;	/* Replaces iod array in fetch. */
+	unsigned int		 nr;	/* number of records in iods    */
```

Comment:
I just made this comment on another patch.   The compiler will add 4 bytes of padding here.  I generally prefer to self pack structs so every field starts on 8 byte boundary.

Reply:
I reordered the members to put the pointers first, the 4-byte int last.